src/main/java/teamdraco/finsandstails/client/model/CrownedHorateeModel.java

```diff
@@ -45,11 +46,9 @@ public void setLivingAnimations(CrownedHorateeEntity entity, Integer uniqueID, A
 			root.setPositionY(-13F);
 		}
 		if (entity.getDeltaMovement().lengthSqr() > 1.0E-7D) {
-			if (entity.isInWater()) {
-				if (entity.isOnGround()) {
-					head.setRotationX(extraData.headPitch * (float) Math.PI / 180F);
-					head.setRotationY(extraData.netHeadYaw * (float) Math.PI / 180F);
-				}
+			if (!entity.isInWater() || entity.isOnGround()) {
+				head.setRotationX(extraData.headPitch * (float) Math.PI / 180F);
+				head.setRotationY(extraData.netHeadYaw * (float) Math.PI / 180F);
 			} else {
 				head.setRotationX(extraData.headPitch * (float) Math.PI / 180F);
 				head.setRotationY(extraData.netHeadYaw * (float) Math.PI / 180F);
```

Comment:
This else isn't necessary. You're saying "if it meets this requirement, do this. If it doesn't, still do it"